docs/agents/sematext-agent/releasenotes.md

```diff
@@ -35,6 +35,28 @@ NULL
 
 -->
 
+## Version 3.1.0
+
+Date: August 24, 2023
+
+### New Features
+
+- Varnish Cache metrics monitoring support: With this update, Sematext Agent can now discover and monitor your Varnish Cache service. Stay tuned for the upcoming release of Varnish Cache monitoring integration in Sematext Cloud.
```

Comment:
I think it's better to replace with: ...Sematext Agent can now discover **and collect metrics** from your ....